src/frontend/Komponenter/Behandling/Vurdering/vurderingValg.ts

```diff
@@ -44,3 +45,10 @@ export const årsakValgTilTekst: Record<ÅrsakOmgjøring, string> = {
     IKKE_UTREDET_NOK: 'Ikke utredet nok',
     ANNET: 'Annet',
 };
+
+export const årsakAlternativer: (fagsystem: Fagsystem) => Record<string, string> = (fagsystem) =>
+    Object.fromEntries(
+        Object.entries(årsakValgTilTekst).filter(
+            ([nøkkel]) => fagsystem !== Fagsystem.EF || nøkkel !== ÅrsakOmgjøring.IKKE_UTREDET_NOK
```

Comment:
Denne burde nok gjøres positiv, negativ logikk og || er vanskelig å lese.

Kanskje trekke ut logikken for at fagsystemet skal være BA eller KS og at nøkkelen skal være `ÅrsakOmgjøring.IKKE_UTREDET_NOK` i en egen const. Og kalle den noe slik som `spesiellNøkkelSomBAKSTrenger`?